plugin-flex-ts-template-v2/src/feature-library/dispositions/custom-components/DispositionTab/DispositionTab.tsx

```diff
@@ -50,23 +72,45 @@ const DispositionTab = (props: OwnProps) => {
     if (isNotesEnabled() && notes) {
       payload.notes = notes;
     }
-
+    payload.custom_attributes = { ...customAttributes };
     dispatch(updateDisposition(payload));
   };
 
   const updateStoreDebounced = debounce(updateStore, 250, { maxWait: 1000 });
```

Comment:
This line is only used within the `useEffect`. You could move this directly into the `useEffect`.